codegen/header.go

```diff
@@ -5,13 +5,17 @@ import (
 )
 
 // Header returns a Go source file header section template.
-func Header(title, pack string, imports []*ImportSpec) *SectionTemplate {
+func Header(title, pack string, imports []*ImportSpec, disableVersion bool) *SectionTemplate {
+	var toolVersion string
+	if !disableVersion {
+		toolVersion = goa.Version()
+	}
 	return &SectionTemplate{
 		Name:   "source-header",
 		Source: headerT,
 		Data: map[string]any{
 			"Title":       title,
-			"ToolVersion": goa.Version(),
+			"ToolVersion": toolVersion,
```

Comment:
Just to be consistent maybe this should be `disableVersion` and the test in the template should be `{{ if not .DisableVersion }}`?